src/MatBlazor/Components/MatTable/MatTable.razor

```diff
@@ -269,18 +270,20 @@
 
     async Task SearchPagedData()
     {
-        var PagedData = await Http.GetJsonAsync<TableItem>(ApiUrl + SearchTermParam(SearchTerm));
-        Items = (IEnumerable<TableItem>)
-            PagedData
-                .GetType()
-                .GetProperty(PagingDataPropertyName)
-                .GetValue(PagedData);
-        var Count = Convert.ToInt32(
-            PagedData
-                .GetType()
-                .GetProperty(PagingRecordsCountPropertyName)
-                .GetValue(PagedData)
-            );
+        var PagedData = await Http.GetStringAsync(ApiUrl + SearchTermParam(SearchTerm));
+        var doc = JsonDocument.Parse(PagedData);
+        var root = doc.RootElement;
+        var itemsString = root.GetProperty(PagingDataPropertyName).GetRawText();
+        
+        var opt = new JsonSerializerOptions()
+        {
+            PropertyNamingPolicy = JsonNamingPolicy.CamelCase
+        };
+        
+        var items = JsonSerializer.Deserialize<List<TableItem>>(itemsString, opt);
+        
+        Items = items;
```

Comment:
why not set Items directly?

Reply:
Hi,
it seams that "change base branch" operations took additional changes from different PR into PR. I am not the author of these changes (may be PR #769). I am not sure if original author will be notified ...